extensions/GitExtension/FileExplorerGitIntegration/Models/GitLocalRepositoryProviderFactory.cs

```diff
@@ -35,10 +34,10 @@ GetLocalRepositoryResult ILocalRepositoryProvider.GetRepository(string rootPath)
         {
             return new GetLocalRepositoryResult(new GitLocalRepository(rootPath, _repositoryCache));
         }
-        catch (RepositoryNotFoundException libGitEx)
+        catch (ArgumentException ex)
         {
-            _log.Error("GitLocalRepositoryProviderFactory", "Failed to create GitLocalRepository", libGitEx);
-            return new GetLocalRepositoryResult(libGitEx, _stringResource.GetLocalized("RepositoryNotFound"), $"Message: {libGitEx.Message} and HRESULT: {libGitEx.HResult}");
+            _log.Error("GitLocalRepositoryProviderFactory", "Failed to create GitLocalRepository", ex);

```

Comment:
I believe the exception needs to come first in .Error, yeah? @krschau

Reply:
Nice catch! Yes, it does need to come first. Thanks!